### PR TITLE
Adds the rorschach mask.

### PR DESCRIPTION
**Pull request by harmonyn**
Kor made me do it

Imgur

:cl:
rscadd: Adds the rorschach mask to the game. It can be found in the AutoDrobe next to the owl suit.
/:cl:

### CODE REVIEW

**Comment by lzimann** [comment]
cool

**Comment by Anonmare** [comment]
Needs a deep fat frier for "grilling" inmates with

**Comment by PKPenguin321** [comment]
you should probably mention in the changelog that it can be found in the autodrobe

**Comment by Incoming5643** [comment]
can we at least pretend to not be violating copyright

**Comment by bgobandit** [comment]
nonsense, it's just a simple mask with some rorschach test inkblots on it

**Comment by Lumipharon** [comment]
^

**Comment by WJohn** [comment]
It's not even animated. Why would anyone wear this?

**Comment by bgobandit** [comment]
what's wrong with the sprites

**Comment by Jalleo** [comment]
If its got the name of the character its pretty much a total rip.

**Comment by Dorsisdwarf** [comment]
It's a mask with a rorsach pattern on it, copyright got shit on it.

On the other hand it should be the "vigilante mask" so its good for more than exactly one joke.

**Comment by WJohn** [comment]
@bgobandit 
The sprites are made of pure white, pure black, and a grey outline that is nearly identical to standard floor tiles. It lacks basic shading, so the contrast is TOO strong for what should be cotton. Also, it's not animated! That's like the coolest part.

**Comment by swankcookie** [comment]
@bgobandit want me to take a look at some respriting? I think at the very least, it needs movement.

**Comment by bgobandit** [comment]
sure go ahead

**Comment by The-Albinobigfoot** [comment]
Can we just stop with the overt pop-culture references? I'm just wondering what the point is exactly - what is the justification for a comic book icon to appear in ss13?

**Comment by PKPenguin321** [comment]
>Oh no, I can't kill people with this! It's shit, don't add it!

**Comment by The-Albinobigfoot** [comment]
Give me one good reason why we need this.

**Comment by PKPenguin321** [comment]
it might allow a player to run a gimmick which creates fun, and there's no harm in not having it.

nice fallacy fallacy by the way :^)

**Comment by The-Albinobigfoot** [comment]
So can we add the raincoat from Psycho? How about the jumpsuit from Kill Bill? Immorten Joe's facemask? What about Magneto's helm? 
If a completely irrelevant comic book character gets to be represented, then why not every single creation from pop-culture? It sets a precedent for future additions that are even more contrived. 

>Invoking the "fallacy" fallacy

You're reaching there, friend. :^)

**Comment by PKPenguin321** [comment]
>If a completely irrelevant comic book character gets to be represented, then why not every single creation from pop-culture?

because nobody's willing to sprite it. if you want them, go ahead and make them. :)

**Comment by The-Albinobigfoot** [comment]
>If someone puts forth a modicum of effort into something, we'll add it no argument.

That attitude is why this server is becoming increasingly bloated with unnecessary features and additions.

**Comment by PKPenguin321** [comment]
oh no this mask is going to trigger the absolute downfall of the server! *inb4 "muh fallacies!"*

cmon man, i dont know what crawled into your ass but you need to not be so antagonistic

**Comment by The-Albinobigfoot** [comment]
If someone is going to add something that is essentially a meme, is it so much to ask that they do it in a non-obtrusive, graceful way? The multi-tool is a reference to Deus Ex; the mining exo "Ripley" is a reference to Aliens; several wizard spell incantations are references; the engie hardsuits are (I think) a Dead Space reference.

Point is - this game does have quite a few shout-outs, but for the most part, they're tounge-in-cheek and done tastefully. My beef is with things like this - all it amounts to is "lol Rorschach". It's thoughtless, derivative, and bereft of an "aha" moment where you realize it's a shout-out. Someone is going to see this and think one of two things at best: either "Oh, Rorschach, that's cool" or "What the fuck is that?" Someone could spend hours on this game not realizing that some of the examples from the previous paragraph are references, but with this, it is simply self-referential, serving no purpose other than to be a joke for the sake of having a joke.

For what's it worth I just think this /tg/ needs to develop an identity, a clear vision - "An X for the sake of having an X" is a problem that's endemic of this server, but that's a complaint for another time I suppose, I'm sure I look autistic as hell, and I've said my piece.

**Comment by PKPenguin321** [comment]
tl;dr

**Comment by The-Albinobigfoot** [comment]
And you have the audacity to tell me to "not be so antagonistic"?

:^)

**Comment by PKPenguin321** [comment]
\>a literal who shits up a PR with asinine, hostile, borderline-autistic complaints and gets offended when i don't read his actual wall of text
i wont be replying further
:ok_hand:

**Comment by The-Albinobigfoot** [comment]
>shits up a PR with asinine, hostile, borderline-autistic complaints

It takes two people to have an argument; if you really thought that a bottom-of-the-totem-pole autist's comment was so shit, and that you're "2cool2post", then why did you even bother yourself to respond with:

>Oh no, I can't kill people with this! It's shit, don't add it!

Could have just ignored me, and nothing would have come of it. But look where we are now.

**Comment by Dorsisdwarf** [comment]
PKP a shit

Albino a shit 

QED

**Comment by swankcookie** [comment]
yeah guys, that was super shit on both your ends.  PKP, maybe add it in a more graceful way. Albino, if you have concerns around how the server is changing, bring it up in a general discussion, rather than in direct response to additions, where you will come off as aggressive or as you both so gracefully put it, "antagonistic".

**Comment by duncathan** [comment]
>For what's it worth I just think this /tg/ needs to develop an identity, a clear vision

This was the part where I realized this guy is fuckin with us

**Comment by swankcookie** [comment]
Thats the issue - i dont think he is.

**Comment by PKPenguin321** [comment]
i think the mask is cool

**Comment by The-Albinobigfoot** [comment]
All I'm going to say is that a prefer my memes with a bit of subtlety. That's all.

**Comment by optimumtact** [comment]
Can people stop with the ridiculous copyright claims?

Ever since the fucking roro's this shit has been way out of control

**Comment by Razharas** [comment]
Ok i pronounce this legit

**Comment by Razharas** [comment]
However WJ is a sprite master and he said sprites are shit so you know

**Comment by Razharas** [comment]
@bgobandit

**Comment by swankcookie** [comment]
@bgobandit im working on some sprites rn. wait on this for a minute

**Comment by bgobandit** [comment]
holy shit this blew up

awaiting sprites senpai

**Comment by Super3222** [comment]
I love this, but by any chance could the mask shuffle inkblot patterns as well?

**Comment by bgobandit** [comment]
@swankcookie how are those sprites coming

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -839,7 +839,7 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/tie/waistcoat = 1,/obj/item/clothing/under/suit_jacket = 1,/obj/item/clothing/head/that =1,/obj/item/clothing/under/kilt = 1,/obj/item/clothing/head/beret = 1,/obj/item/clothing/tie/waistcoat = 1,
 					/obj/item/clothing/glasses/monocle =1,/obj/item/clothing/head/bowler = 1,/obj/item/weapon/cane = 1,/obj/item/clothing/under/sl_suit = 1,
 					/obj/item/clothing/mask/fakemoustache = 1,/obj/item/clothing/suit/bio_suit/plaguedoctorsuit = 1,/obj/item/clothing/head/plaguedoctorhat = 1,/obj/item/clothing/mask/gas/plaguedoctor = 1,
-					/obj/item/clothing/suit/toggle/owlwings = 1, /obj/item/clothing/under/owl = 1,/obj/item/clothing/mask/gas/owl_mask = 1,
+					/obj/item/clothing/mask/rorschach = 1, /obj/item/clothing/suit/toggle/owlwings = 1, /obj/item/clothing/under/owl = 1,/obj/item/clothing/mask/gas/owl_mask = 1,
```
Comment (phil235) [inline code/game/machinery/vending.dm]: please read https://github.com/tgstation/-tg-station/pull/15290/files#diff-be8fc4ef536d83a7c390019dd59d14efR601

Comment (bgobandit) [inline code/game/machinery/vending.dm]: what

Comment (duncathan) [inline code/game/machinery/vending.dm]: I'm uh
Not seeing the point here @phil235

Comment (phil235) [inline code/game/machinery/vending.dm]: sorry, I mean line 601 of this file. You need to update the amount of the autodrobe resupply unit.

**Comment by KorPhaeron** [comment]
@bgobandit 

Let me know if you ever get sprites

**Comment by bgobandit** [comment]
@swankcookie
